_layouts/post.html

```diff
@@ -12,47 +12,44 @@
+      <div id="toc">
+          {% toc %} {{page.document | tocify_asciidoc}}
+      </div>
     </div>
     <div class="col-md-7 post-section-white">
-        <div class="col-md-9 col-md-offset-2 post-title text-center">
-            <img src="{{ page.image }}" alt="post logo">
-            <label>{{ page.title }}</label>
-            <p>
-                {% assign words = content | number_of_words %}
-                {% if words < 360 %}
-                  1 min read.
-                {% else %}
-                  {{ words | divided_by:180 }} mins read.
-                {% endif %} 
-                Updated {{ page.date | date_to_string }}
-            </p>
-            {% if page.tags %}
-              {% for tag in page.tags %}
-                <span class="badge badge-pill badge-default">
-                  {% capture tag-name %}{{tag | upcase }}{% endcapture %}
-                  {% if tag-name=="AWS" or tag-name=="GCP"  or tag-name=="GKE" %} {{ tag-name }} {% else %} {{tag-name | capitalize}}{% endif %}</span>
-              {% endfor %}
-            {% endif %}
-          </div>
-          <div class="content">
-              {{ content }}
-          </div>
-
+      <div class="col-md-9 col-md-offset-2 post-title text-center">
+        <img src="{{ page.image }}" alt="post logo">
+        <label>{{ page.title }}</label>
+        <p>
+          {% assign words = content | number_of_words %}
+          {% if words < 360 %}
+            1 min read.
+          {% else %}
+            {{ words | divided_by:180 }} mins read.
+          {% endif %} 
+          Updated {{ page.date | date_to_string }}
+        </p>
+        {% if page.tags %}
+          {% for tag in page.tags %}
+            <span class="badge badge-pill badge-default">
+              {% capture tag-name %}{{tag | upcase }}{% endcapture %}
+              {% if tag-name=="AWS" or tag-name=="GCP"  or tag-name=="GKE" %} {{ tag-name }} {% else %} {{tag-name | capitalize}}{% endif %}</span>
+          {% endfor %}
+        {% endif %}
+      </div>
+      <div class="content">
+          {{ content }}
+      </div>
     </div>
     <div class="col-md-2">
-        <div class="card contact-us-card text-center">
-          <span class="glyphicon glyphicon-exclamation-sign" aria-hidden="true"></span>
-            <div class="card-body">
-              <p class="card-title">Want to learn more? Feel free to contact us for support</p>
-              <a href="#" class="btn btn-primary">Support</a>
-              <a href="#">Dismiss</a>
-            </div>
-          </div>
-    </div>
-    </div>
+      <div class="card contact-us-card text-center">
+        <span class="glyphicon glyphicon-exclamation-sign" aria-hidden="true"></span>
+        <div class="card-body">
+          <p class="card-title">Want to learn more? Feel free to contact us for support</p>
+          <a href="#" class="btn btn-primary">Support</a>
+          <a href="#">Dismiss</a>
+        </div>
+      </div>
```

Comment:
Are there real changes on this page or just whitespace changes? Quite hard to tell. If it's whitespace, could you tweak your editor to avoid that, so PRs are easier to review (and diffs easier to use for debugging)?

Reply:
Yeah, these are just white spaces. Both changes were made by me as this branch was checked out of the [translate-deployment-guides](https://github.com/gruntwork-io/gruntwork-io.github.io/tree/translate-deployment-guides)